OracleSOA/README.md

```diff
@@ -0,0 +1,77 @@
+UPDATE: Move to [jeqo/oracle-soa-docker](https://github.com/jeqo/oracle-soa-docker)
```

Comment:
I followed this link and found images on Altas and the Docker Hub. **These images must be removed immediately from both locations** as they breach the terms and conditions of Oracle's software license. 

You cannot publish Oracle binaries on the Docker Hub, which includes all of Oracle Java, WebLogic and the SOA suite.